run_unittests.py

```diff
@@ -1815,6 +1815,32 @@ def test_order_of_l_arguments(self):
                     return
         raise RuntimeError('Linker entries not found in the Ninja file.')
 
+    def test_introspect_dependencies(self):
+        '''
+        Tests that mesonintrospect --dependencies returns expected output.
+        '''
+        if not shutil.which('pkg-config'):
+            raise unittest.SkipTest('pkg-config not found')
+        if subprocess.call(['pkg-config', '--exists', 'glib-2.0', 'gobject-2.0']) != 0:
+            raise unittest.SkipTest('GLib and GObject not found with pkg-config')
```

Comment:
I looked at what LinuxlikeTests.test_qt5dependency_pkgconfig_detection() does. Qt is not a requirement but GTK+ is?